1-first-unix-commands.sh

```diff
@@ -0,0 +1,38 @@
+# q1
+# first approach
+echo "damian" > /home/damianti/firstFile.txt
+# second approach
+touch /home/damianti/secondfile.txt
+echo "damian" >> /home/damianti/secondfile.txt
+
+
+# q2
+mkdir /home/damianti/new
```

Comment:
Instead of the full path to the home directory, it's advised to use the `~` alias